page_creation.rst

```diff
@@ -116,9 +117,14 @@ First, make sure that ``LuckyController`` extends Symfony's base
 Now, use the handy ``render()`` function to render a template. Pass it our ``number``
 variable so we can render that::
 
+    <?php
     // src/AppBundle/Controller/LuckyController.php
+    namespace AppBundle\Controller;
+
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-    // ...
```

Comment:
We use this placeholder comment in many places to indicate that the code example is not complete. This is usually done when the code evolves during the reader following an article down from the top. We then just try to add what is new for this specific example.

In this file, the `LuckyController` class was already introduced before (in the other code example that you changed) including the needed namespace and `use` statements.

Furthermore, we omit the opening `<?php` tags everywhere for brevity to keep all code examples as small as possible.

Thus, for me there is nothing to change here. Anyway, thank you very much for opening this PR. We really appreciate people coming with feedback and trying to improve the docs.